api/api/models_api.go

```diff
@@ -17,6 +17,9 @@ package api
 import (
 	"fmt"
 	"net/http"
+	"strconv"
+
+	"github.com/caraml-dev/merlin/service"
```

Comment:
Nit. The convention with ordering imports is:
* Standard packages
* Public packages, Like `"github.com/jinzhu/gorm"`
* Local packages. Here, `"github.com/caraml-dev/merlin/service"` is a local package, for this app.

Reply:
i was using the goimports package to sort the import, let me change it 👍